webapp/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```

Comment:
Remove please. We can update READEME later but Let's not have generated ones commit to the master branch.